proxies.py

```diff
@@ -0,0 +1,183 @@
+"""
```

Comment:
This and authd.py should probably be combined into a file that could be named something related to handlers.